doc/source/tune/tutorials/tune-resources.rst

```diff
@@ -18,6 +18,10 @@ of CPUs (cores) on your machine.
     )
     results = tuner.fit()
 
+.. warning::
```

Comment:
Can we move this warning below the next paragraph? We should introduce `tune.with_resources` before having this warning.